molo/core/templates/base.html

```diff
@@ -27,6 +27,9 @@
     <meta property="og:image" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
     <meta property="og:image:url" content="{% if self.social_media_image %}{{request.site.root_url}}{{ tmp_photo.url }}{% endif %}"/>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    {% if settings.core.SiteSettings.google_search_console %}
+    <meta name="google-site-verification" content="{{settings.core.SiteSettings.global_google_search_console}}" />
```

Comment:
In the If statement you check for `.google_search_console` but here you're using `.global_google_search_console` what's the difference?

Reply:
Hey, I seem to have made a mistake in the naming. 
Thank you